.agents/skills/ansible-testing-linting/SKILL.md

```
@@ -0,0 +1,63 @@
+## Validation Commands
+
+You must run the following checks on playbooks whenever any YAML configs or tasks are changed.
+
+### 1. Playbook Syntax Verification
+Ensure there are no syntax errors in your Ansible YAML playbooks:
+```bash
+ansible-playbook --syntax-check play.yml
+ansible-playbook --syntax-check play-localhost.yml
+```
+
+### 2. Playbook Style & Lint Checking
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)